docker-compose.yml

```diff
@@ -1,27 +1,11 @@
 version: "3"
 services:
-  mysql:
-    image: mariadb:10.1
-    environment:
-      MYSQL_ROOT_PASSWORD: supersecret
-      MYSQL_USER: powerdns
-      MYSQL_PASSWORD: secret
-      MYSQL_DATABASE: powerdns
-    volumes:
-      - database:/var/lib/mysql
-    networks:
-      - backend
   powerdns:
-    image: psitrax/powerdns
-    environment: 
-      MYSQL_HOST: mysql
-      MYSQL_USER: powerdns
-      MYSQL_PASS: secret
-      MYSQL_DB: powerdns
+    image: powerdns/pdns-auth-46:4.6.3
+    environment:
+      PDNS_AUTH_API_KEY: secret
     networks:
       - backend
-    volumes:
-      - ./.docker/pdns:/etc/pdns/conf.d
     ports:
       - 8081:8081
 volumes:
```

Comment:
Since this will be stateless after this change, we can remove the networks and volumes as well, right?